google/ggrc-core
# GGRC-8062 Blue flag icon for Program doesn't appear after mapping Child program until the page is reloaded

# Issue description

Blue flag icon for Program doesn't appear after mapping Child program until the page is reloaded

# Steps to test the changes

1. Create 2 programs.
2. Open a program 1 and click Add tab.
3. Click Child Programs object.
4. Select a program 2 and Map as Child.
5. Click Proceed button on confirmation popup.
6. Expected: Blue flag icon displays on info page of program.

# Solution description

Added event handler for check when instance has been updated.

# Sanity checklist




# PR Review checklist
